x-pack/plugins/ml/public/file_datavisualizer/components/analysis_summary/analysis_summary.js

```diff
@@ -32,22 +34,26 @@ export function AnalysisSummary({ results }) {
 
       <EuiDescriptionList
         type="column"
-        listItems={items}
+        listItems={itemsWithI18n}
         className="analysis-summary-list"
+        titleProps={{ tabIndex: '0' }}
       />
     </React.Fragment>
   );
 }
 
+function pimpOutDisplayItemsWithAriaAndI18n(items) {
+  return items.map(item => Object.assign(item, {
```

Comment:
this has the side effect of modifying the original `items` array.
Instead the item should be copied and `title` overwritten. e.g.
```
return items.map(item => (
    {
      ...item,
      title: <span aria-label={`${item.title}: ${item.description}`}>{item.title}</span>
    }
  ));
```